pkgs/applications/radio/qdmr/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  writeText,
+  cmake,
+  wrapQtAppsHook,
+  libusb1,
+  libyamlcpp,
+  qtlocation,
+  qtserialport,
+  qttools,
+  qtbase,
+}:
+stdenv.mkDerivation rec {
+  pname = "qdmr";
+  version = "0.11.1";
+
+  src = fetchFromGitHub {
+    owner = "hmatuschek";
+    repo = "qdmr";
+    rev = "v${version}";
+    sha256 = "sha256-nW7ha8JB6aHRegHOc7gTM+JGyEH181MYj/oto1gkd/U=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libyamlcpp
+    libusb1
+    qtlocation
+    qtserialport
+    qttools
+    qtbase
+  ];
+
+  postInstall = ''
+    mkdir -p "$out/etc/udev/rules.d"
+    cp ${src}/dist/99-qdmr.rules $out/etc/udev/rules.d/
+  '';
+
+  desktopItems = [
```

Comment:
You are using a `desktopItems`; so, you need `copyDesktopItems` inside `nativeBuildInputs`.

Reply:
Added it in the latest commit